Wikipedia/Localizations/en.lproj/Localizable.strings

```diff
@@ -332,3 +332,28 @@
 "home-nearby-footer" = "More from nearby your location";
 "home-nearby-nothing" = "Nothing nearby";
 "home-nearby-check-again" = "Check again";
+
+"welcome-whats-new-title" = "Welcome To Wikipedia";
+"welcome-whats-new-whats-new" = "WHAT'S NEW";
```

Comment:
Have gotten push-back from TWN in past for using all caps - they said this made translation to certain languages trickier and to use normal casing. If we need all caps we can then upper case just before display for English only.